app/controllers/bookingsync_portal/admin/remote_accounts_controller.rb

```diff
@@ -18,6 +19,11 @@ def scope
         current_account.remote_accounts
       end
 
+      def synchronize_bookingsync_source_id
+        source = BookingsyncPortal::Write::Source.new(current_account).synchronize
+        current_account.update(synced_source_id: source.id)
```

Comment:
I would make it a bang method to make sure it really stores the source id, not fails silently due to potential validation errors.